### DISCUSSION TITLE
Problém with time based revalidation

### DISCUSSION THREAD

**Discussion by pjanickovic-ph** [vercel/next.js#70326] [Help]
I have a dashboard app, all component have a Server side async part, some of them have a client (chart or so). I am reading data from redis. (Not using fetch at all).

In redis, data are updated every 5 min. My page has const export revalidate  = 60, and on client side I am calling router.refresh() every 60 secobds.

The problem is I can see stale data even after 15 min or so. Only things what helps is F5

Any suggestion?

**Comment by icyJoseph** [comment]
Here, https://github.com/vercel/next.js/discussions/54075, router.refresh is used to do that. 

What Next.js version have you got?
